packages/queued-request-controller/src/QueuedRequestController.ts

```diff
@@ -168,61 +245,87 @@ export class QueuedRequestController extends BaseController<
 
     await this.messagingSystem.call(
       'NetworkController:setActiveNetwork',
-      request.networkClientId,
+      originNetworkClientId,
     );
   }
 
-  #updateCount(change: -1 | 1) {
+  /**
+   * Update the queued request count.
+   */
+  #updateCount() {
     this.update((state) => {
-      state.queuedRequestCount += change;
+      state.queuedRequestCount = this.#requestQueue.length;
```

Comment:
is this now counting the number of batches? or total number of requests? not sure if it matters or how this is used?

Reply:
ok I think I see its the latter